setup.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
-import os, pcpp
```

Comment:
If I'm trying to pip install pcpp directly from the internet it can't read this file as importing pcpp fails due to ply missing - it can't get far enough down this file to know that ply is a declared dependency.